src/main/java/com/squareup/javapoet/CodeBlock.java

```diff
@@ -269,6 +278,13 @@ public Builder addStatement(String format, Object... args) {
       return this;
     }
 
+    public Builder addStatement(CodeBlock codeBlock) {
```

Comment:
I don't understand why you would ever need this -- you could just call addStatement on the codeBlock argument.